### PR TITLE
schema: add submissionTerms.electronicSubmissionPolicy to tender

### LINKED ISSUE #1183: Merge the submissionTerms extension
> We can consider merging the submission terms extension into core. That extension supports the only clear use case from the discussion in #322, which is distinguishing opportunities for which electronic submission is required/allowed/not allowed.

https://github.com/open-contracting/standard/issues/322#issuecomment-763810724

[submissionTerms extension](https://github.com/open-contracting-extensions/ocds_submissionTerms_extension)

### PR DESCRIPTION
**Pull request by odscjen**
closes #1183 as discussed, i.e. moved `submissionTerms.electronicSubmissionPolicy` across to `tender` only

### CODE REVIEW

**Review by duncandewhurst — APPROVED** [review]

File: schema/release-schema.json [hunk 1]
```diff
@@ -394,6 +394,29 @@
             "null"
           ]
         },
+        "submissionTerms": {
+          "title": "Submission terms",
+          "description": "Information about the submission terms to describe how, when and where the tenderers need to submit their bids for the procedure.",
```
Comment (jpmckinney) [inline schema/release-schema.json]:
```suggestion
          "description": "Information about how, when and where tenderers need to submit their bids.",
```

**Review by jpmckinney — COMMENTED** [review]
Avoiding a little circularity in the definition.

**Comment by odscjen** [comment]
@jpmckinney all I did was commit your suggestion and the tests are now failing with a url 404 can't find "https://raw.githubusercontent.com/open-contracting/standard/1183_add_electronicSubmissionPolicy/schema/record-schema.json"? The tests pass on my local machine...

**Review by jpmckinney — APPROVED** [review]

**Comment by jpmckinney** [comment]
Had to merge 1.2-dev, and also run manage.py pre-commit.

**Pull request merged**